backend/medtagger/api/scans/business.py

```diff
@@ -8,6 +8,7 @@
 from medtagger.types import ScanID, LabelPosition, LabelShape, LabelSelectionBinaryMask, ScanMetadata, LabelingTime
 from medtagger.database.models import ScanCategory, Scan, Slice, Label, SliceOrientation
 from medtagger.repositories.labels import LabelsRepository
+from medtagger.api.users.business import get_current_user_info
```

Comment:
Please avoid importing one business logic to another one. It will result in bad code structure and easy to create circular imports.